### ISSUE TITLE
Question about ROS Message Frame 3D Points

### ISSUE THREAD

**Issue by epruc** [ravijo/ros_openpose#91]
I just wanted to clarify the components of the ros messgae frame. Specifically the geometry_msgs/Point32 point my assumption is that these are the 3D coordinates of the skeleton points detected by openpose in the x,y,z world coordinate frame and that these points are ordered the same way as openpose, namely 

            { 0,      "Nose"},    {13,      "LKnee"}
            { 1,      "Neck"},    {14,     "LAnkle"}
            { 2, "RShoulder"},    {15,       "REye"}
            { 3,    "RElbow"},    {16,       "LEye"}
            { 4,    "RWrist"},    {17,       "REar"}
            { 5, "LShoulder"},    {18,       "LEar"}
            { 6,    "LElbow"},    {19,    "LBigToe"}
            { 7,    "LWrist"},    {20,  "LSmallToe"}
            { 8,    "MidHip"},    {21,      "LHeel"}
            { 9,      "RHip"},    {22,    "RBigToe"}
            {10,     "RKnee"},    {23,  "RSmallToe"}
            {11,    "RAnkle"},    {24,      "RHeel"}
            {12,      "LHip"},    {25, "Background"}

Is this correct?

**Comment by ravijo** [comment]
@epruc 

> Is this correct?

Yes. It is correct for the `BODY_25` model used in Openpose. The output is utilized for visualization, and these IDs are assigned as shown below:

https://github.com/ravijo/ros_openpose/blob/21e6c3e23bd49340e04aee1341e894142d9f66e7/scripts/visualizer.py#L46-L69


Hope it helps!
